sdk/storage/storage-blob/src/IBlobSASSignatureValues.ts

```diff
@@ -372,6 +547,109 @@ function generateBlobSASQueryParameters20181109(
   );
 }
 
+/**
+ * ONLY AVAILABLE IN NODE.JS RUNTIME.
+ * IMPLEMENTATION FOR API VERSION FROM 2018-11-09.
+ *
+ * Creates an instance of SASQueryParameters.
+ *
+ * Only accepts required settings needed to create a SAS. For optional settings please
+ * set corresponding properties directly, such as permissions, startTime and identifier.
+ *
+ * WARNING: identifier will be ignored, permissions and expiryTime are required.
+ *
+ * @param {IBlobSASSignatureValues} blobSASSignatureValues
+ * @param {UserDelegationKeyCredential} userDelegationKeyCredential
+ * @returns {SASQueryParameters}
+ */
+function generateBlobSASQueryParametersUDK20181109(
+  blobSASSignatureValues: IBlobSASSignatureValues,
+  userDelegationKeyCredential: UserDelegationKeyCredential
+): SASQueryParameters {
+  if (!blobSASSignatureValues.permissions || !blobSASSignatureValues.expiryTime) {
+    throw new RangeError(
```

Comment:
Minor: why this is RangeError?

Reply:
The RangeError object indicates an error when a value is not in the set or range of allowed values.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RangeError